### PR TITLE
errors: Store details of all errors when checking

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-process/issues/301

### CODE REVIEW

**Comment by odscjames** [comment]
@jpmckinney Before I do final checks, does this basically look good?

**Comment by odscjames** [comment]
[kf-process: Catch CoVE errors that aren't API exceptions](https://trello.com/c/gyTXFIUt/232-kf-process-catch-cove-errors-that-arent-api-exceptions)

File: ocdskingfisherprocess/checks.py [hunk 1]
```diff
@@ -186,13 +187,24 @@ def _check_release_row(self, release_row, override_schema_version=''):
...
+            # We save it to the database
+            self._store_release_row_error(release_row, override_schema_version, err)
+        except Exception as err:
+            # This is any old exception
+            # We save it to the database
+            self._store_release_row_error(release_row, override_schema_version, err)
+            # But lets also report this to Sentry, as it may be a system/host error that we should be alerted about
+            capture_exception(err)
+
+    def _store_release_row_error(self, release_row, override_schema_version, err):
```
Comment (jpmckinney) [inline ocdskingfisherprocess/checks.py:200]: I'd put this next to the `_store_record_row_error` method, for easier comparison when reading or refactoring.

Comment (odscjames) [reply ocdskingfisherprocess/checks.py]: Done

**Pull request merged**